spec/utilsSpec.js

```diff
@@ -171,9 +171,10 @@ describe('Utility module tests', () => {
     firstItemLink = 'OppiabotTestActivitiesModel'.link(
       firstModelFileObj.blob_url
     );
-    let secondItemLink = 'OppiabotSnapshotContentModel, OppiabotSnapshotTestingModel'.link(
-      secondModelFileObj.blob_url
-    );
+    let secondItemLink =
+      'OppiabotSnapshotContentModel, OppiabotSnapshotTestingModel'.link(
+        secondModelFileObj.blob_url
+      );
```

Comment:
```suggestion
    let secondItemLink = (
      'OppiabotSnapshotContentModel, OppiabotSnapshotTestingModel'.link(
        secondModelFileObj.blob_url
      )
    );
```

Reply:
Why this change?
It could be:
```
let secondItemLink =
      'OppiabotSnapshotContentModel, OppiabotSnapshotTestingModel'
        .link(secondModelFileObj.blob_url);
```

Reply:
Yeah, I guess this is also fine, I prefer to use (even when the second `()` are not needed). I think the final decision is up to you.

```typescript
let secondItemLink = (
  'OppiabotSnapshotContentModel, OppiabotSnapshotTestingModel'
    .link(secondModelFileObj.blob_url);
);
```